tools/coqdoc/cpretty.mll

```diff
@@ -26,9 +27,9 @@
     let n = String.length s in
     let rec count c i =
       if i == n then c,i else match s.[i] with
-	| '\t' -> count (c + (8 - (c mod 8))) (i + 1)
-	| ' ' -> count (c + 1) (i + 1)
-	| _ -> c,i
+  | '\t' -> count (c + (8 - (c mod 8))) (i + 1)
```

Comment:
What is going on here with the indentation?

Reply:
Fixed.

Reply:
I should have been more clear. The entire file has strange indentation changes.

Reply:
All fixed, modulo tabs versus spaces.